src/plugins/intel_cpu/src/nodes/executors/subgraph.hpp

```diff
@@ -12,6 +12,25 @@
 
 namespace ov::intel_cpu {
 
+namespace {
+// <original ptr idx, postprocessed ptr idx>
+using PtrMapping = std::pair<size_t, size_t>;
```

Comment:
It makes sense to create small structure with two fields with specific names? It should improve readability 
As far as I know, we should try to avoid using `std::pair` due to unnamed fields `first` and `second`

Reply:
I created a separate struct and moved all the ptr mappings related logic to a separate base `SubgraphSpecializedBaseExecutor` class to avoid duplication in the inherited classes for static and dynamic cases.